CHANGELOG.md

```diff
@@ -7,16 +7,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Added
-- `ScmpArgCompare::new` is now a `const fn`.
+- `ScmpVersion::current()` as rustified replacement for `get_library_version()`.
+- `ScmpFilterContext::get_no_new_privs_bit()` to query the state of the No New Privileges bit.
+- `ScmpArch::native()` as rustified replacement for `get_native_arch()`.
+- `ScmpFilterContext::as_ptr()` to return a raw pointer to the `scmp_filter_ctx`.
+- `scmp_cmp!`macro to create a `ScmpArgCompare` in a more elegant way.
+- `impl From<&ScmpArgCompare> for scmp_arg_cmp`.
+- `ScmpFilterContext::set_syscall_priority()` to set the priority of a given syscall.
+- `ScmpFilterContext::add_rule_conditional()` to add a single rule for a conditional
+action on a syscall.
+- `ScmpFilterContext::add_rule_exact()` to add a single rule for an unconditional
+action on a syscall.
+- `ScmpFilterContext::add_rule_conditional_exact()`to add a single rule for a conditional
+action on a syscall.
+- `check_version()` to check that the libseccomp version being used is equal to
+or greater than the specified version.
+- `check_api()` to check that both the libseccomp API level and the libseccomp
+version being used are queal to or greater than the specified API level and version.
 
 ### Changed
+- `get_syscall_name_from_arch` and `get_syscall_from_name` output a syscall number with
+an error message when the functions cannot resolve the syscall name.
+- `ScmpAction::Trace` now holds an `u16` since you can not use any more bits anyway
+(**Incompatible change**).
 - `ScmpAction::Errno` now holds an `i32` to make `ScmpAction::Errno(libc::EPERM)`
-  work without casting.
-- `ScmpAction::Trace` now holds an `u16` since you can not use any more bits anyway.
+  work without casting (**Incompatible change**).
+- `ScmpFilterContext::add_rule()` returns an error if the length of comparators is bigger than `u32`.
```

Comment:
```suggestion
```

After #66 not explicit via try_into but libseccomp will fail anyway if you want to compare more then 4294967295 arguments.

Reply:
Ok. Thank you. I removed it.